docs/pages/guides/sharing-before-shipping.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Sharing your app before you ship
+---
+
+Ultimately, you will want to publish your application to the App Store and Play Store to access the opportunities for wider distribution and to provide a familiar installation process for your users.  While you are developing though, you'll want to let other members of your team and your potential users see the work in progress to gather feedback and make sure you're building a quality product.
```

Comment:
"and your potential users" is toeing the line imo, maybe "and others with a vested interest in your project"?